packages/web/src/components/gcds-select/readme.md

```diff
@@ -24,13 +24,13 @@
 
 ## Events
 
-| Event              | Description                                     | Type                  |
-| ------------------ | ----------------------------------------------- | --------------------- |
-| `gcdsBlur`         | Emitted when the select loses focus.            | `CustomEvent<void>`   |
-| `gcdsError`        | Emitted when the select has a validation error. | `CustomEvent<object>` |
-| `gcdsFocus`        | Emitted when the select has focus.              | `CustomEvent<void>`   |
-| `gcdsSelectChange` | Update value based on user selection.           | `CustomEvent<any>`    |
-| `gcdsValid`        | Emitted when the select has a validation error. | `CustomEvent<object>` |
+| Event        | Description                                     | Type                  |
+| ------------ | ----------------------------------------------- | --------------------- |
+| `gcdsBlur`   | Emitted when the select loses focus.            | `CustomEvent<void>`   |
+| `gcdsChange` | Update value based on user selection.           | `CustomEvent<any>`    |
```

Comment:
```suggestion
| `gcdsChange` | Update value based on user selection.           | `CustomEvent<any>`    |
```
Could this be "Emitted when the select value has changed."? Just trying to match it with the other descriptions.